### PR TITLE
Docs: Removed redundant files and moved Readme.md

### LINKED ISSUE #116: Remove .md file under `docs`
The documentation files `docs/config.md` and `docs/User_modeling.md` are redundant and should be removed.
Move `docs/source/README.md` under `docs`.

### PR DESCRIPTION
**Pull request by adityakode**
fixes #116 
Removed Redundand files `docs/config.md` and `docs/User_modeling.md`.
Moved  `docs/source/README.md` under `docs`.

### CODE REVIEW

**Review by NoB0 — CHANGES_REQUESTED** [review]
Please update the section `User modeling` in the file `docs/source/component.rst` with the content from `docs/User_modeling.md`. 
Guidelines for `.rst` are available [here](https://github.com/iai-group/guidelines/blob/main/restructuredtext/README.md).

**Comment by adityakode** [comment]
in `docs/source`  , file [user_modeling.rst](https://github.com/iai-group/UserSimCRS/blob/main/docs/source/user_modeling.rst) is present . Should I delete that too ?
And in `docs/source/component.rst` do I overwrite the user modelling section with content from `docs/User_modeling.md` or should I append it ?

**Comment by adityakode** [comment]
hi @NoB0  I have made the changes.
PS review. 
Can you tell me ps why tests are failing ? so that I can make amends

File: docs/source/components.rst [hunk 1]
```diff
@@ -21,6 +21,21 @@ User modeling
 
 User modeling consists of three sub-components: preference model, context model, and persona. The responsibility of the preference model is to capture users’ individual tastes and thus allow for a personalized experience. Following Zhang and Balog [4]_, it is modeled as a personal knowledge graph, where nodes can indicate either an item or an attribute. Novel to our work is the modeling of persona, which is used to capture user-specific traits, e.g., user cooperativeness, and context, which can characterize the situation of the user.
```
Comment (NoB0) [inline docs/source/components.rst:22]:
```suggestion
User modeling consists of three sub-components: preference model, context model, and persona.
```

File: docs/source/components.rst [hunk 2]
```diff
@@ -21,6 +21,21 @@ User modeling
...
+
+Context model
+^^^^^^^^^^^^^
+
+In addition to preference and interaction modeling, we also model other user contexts, specifically temporal and relational. Temporal context refers to time context such as *time of the day* and whether it is a *weekday* or *weekend*. Relational context on the other hand is used to indicate the group setting of the user.
+
+Persona
+^^^^^^^
+
+Persona contains user variables, e.g., cooperativeness.
```
Comment (NoB0) [inline docs/source/components.rst:37]:
```suggestion
Persona is used to capture user-specific traits, e.g., cooperativeness.
```

**Review by NoB0 — APPROVED** [review]
LGTM. I made some suggestions to avoid redundancy.

**Comment by NoB0** [comment]
The tests are failing because we are doing some breaking changes. For this PR you do not have to worry about the tests.

**Comment by NoB0** [comment]
@adityakode could you please solve the conflicts with the main branch so I can merge the changes.

**Comment by NoB0** [comment]
Closing this PR as the merge conflicts are resolved in PR #143

**Pull request closed without merging**